pkgs/applications/terminal-emulators/kitty/default.nix

```diff
@@ -67,6 +70,7 @@ buildPythonApplication rec {
     sphinx-copybutton
     sphinxext-opengraph
     sphinx-inline-tabs
+    go
```

Comment:
Where do we use go when we build to module separate?

Reply:
`go-modules.nix` is only fetching the go modules for use by the kitty build setup. The go binary is called from this setup.py: https://github.com/kovidgoyal/kitty/blob/v0.27.0/setup.py#L910